audit/src/test/resources/json/event/TaskCompletedEvent.json

```diff
@@ -1,16 +1 @@
-{
-    "className": "com.ritense.valtimo.contract.event.TaskCompletedEvent",
-    "id": "edb1a672-4ba1-4e79-a5ee-b9658c55fe52",
-    "origin": "somewhere",
-    "occurredOn": "2019-03-18T14:17:11.639114",
-    "createdOn": "2019-03-18T14:17:11.639114",
-    "user": "somebody",
-    "assignee": "test@test.com",
-    "taskId": "taskId",
-    "taskName": "aTaskName",
-    "processDefinitionId": "aProcessDefinitionId",
-    "processInstanceId": "aProcessInstanceId",
-    "variables": {
-        "Key": "Value"
-    }
-}
\ No newline at end of file
+{"className":"com.ritense.valtimo.contract.event.TaskCompletedEvent","id":"edb1a672-4ba1-4e79-a5ee-b9658c55fe52","origin":"somewhere","occurredOn":"2019-03-18T14:17:11.639114","user":"somebody","assignee":"test@test.com","createdOn":"2019-03-18T14:17:11.639114","taskId":"taskId","taskName":"aTaskName","processDefinitionId":"aProcessDefinitionId","processInstanceId":"aProcessInstanceId","variables":{"Key":"Value"}}
```

Comment:
Not sure why the formatting was removed here. Makes it a lot less readable.

Reply:
I removed all the newlines and whitespaces because the json string didn't match the serialized object any more after the required assertion changes.